src/System.Runtime.Extensions/System.Runtime.Extensions.sln

```diff
@@ -75,8 +78,8 @@ Global
 		{9F312D76-9AF1-4E90-B3B0-815A1EC6C346}.Debug|Any CPU.Build.0 = netstandard-Debug|Any CPU
 		{9F312D76-9AF1-4E90-B3B0-815A1EC6C346}.Release|Any CPU.ActiveCfg = netstandard-Release|Any CPU
 		{9F312D76-9AF1-4E90-B3B0-815A1EC6C346}.Release|Any CPU.Build.0 = netstandard-Release|Any CPU
-		{845D2B72-D8A4-42E5-9BE9-17639EC4FC1A}.Debug|Any CPU.ActiveCfg = netcoreapp-Windows_NT-Debug|Any CPU
-		{845D2B72-D8A4-42E5-9BE9-17639EC4FC1A}.Debug|Any CPU.Build.0 = netcoreapp-Windows_NT-Debug|Any CPU
+		{845D2B72-D8A4-42E5-9BE9-17639EC4FC1A}.Debug|Any CPU.ActiveCfg = uapaot-Windows_NT-Debug|Any CPU
```

Comment:
yeah, this one shouldn't be here as the active configuration, only on your local development.

---
In reply to: [129703878](https://github.com/dotnet/corefx/pull/22664#discussion_r129703878) [](ancestors = 129703878)